mgmt-lambda/handlers/updateHandler.ts

```diff
@@ -66,6 +67,22 @@ export async function handleUpdate(
   }
 
   const newVersion = newVersions[0]
+  const newVersionCodeSha256 = newVersion.CodeSha256
+
+  if (currentCodeSha256 === newVersionCodeSha256) {
+    console.info(`New version's code SHA256 is equal to the previous $LATEST code SHA256`)
+    return {
+      statusCode: 200,
+      body: JSON.stringify({ status: 'Update is not required' }),
+      headers: {
+        'content-type': 'application/json',
+      },
+    }
+  } else {
```

Comment:
We can remove `else` to improve readability, and just have

```
console.info(
      `New version's SHA256 is not equals to the previous one. Continue with updating CloudFront resources...`
    )
```

without `else`.